test/test-normandy-arguments.ts

```diff
@@ -1,26 +1,27 @@
 #!/usr/bin/env ts-node-script
-import { promises as fs } from "fs";
 
 import fetch from "node-fetch";
-import Ajv from "ajv";
 
-const NORMANDY_API_BASE = "https://normandy.cdn.mozilla.net/api";
-
-async function main() {
-  const ajv = new Ajv();
+import { typeGuards } from "..";
 
-  for await (const recipe of fetchEnabledRecipes()) {
-    const revision = recipe.approved_revision ?? recipe.latest_revision;
-    const typeName = convertActionNameToTypeName(revision.action.name);
-    const schema = await fs.readFile(`./schemas/normandy/${typeName}.json`);
+const NORMANDY_API_BASE = "https://normandy.cdn.mozilla.net/api";
```

Comment:
Silly Q: but does this need to be an env var or anything, or will we always be fetching from the production normandy API server?

Reply:
The goal here is not to test that the Normandy server works, but to get a sample of real world data. Because of that I don't think we'll need to be changing it often enough to warrant using an envvar.